include/rdma/fi_domain.h

```diff
@@ -103,6 +103,7 @@ struct fi_mr_attr {
 	uint64_t		offset;
 	uint64_t		requested_key;
 	void			*context;
+	size_t			auth_key_index;
```

Comment:
I didn't see one, but I think we should discuss.  It seems more burdensome in some cases.